ansible/provision_aws.yml

```diff
@@ -64,26 +64,39 @@
             to_port: all
             cidr_ip: 0.0.0.0/0
         vpc_id: "{{ ecs_vpc_id }}"
+      register: sg
```

Comment:
let's call this one `ecs_sg` for consistency with the `elb_sg` used by the load balancer above.